Ameba 0.9.2 doesn't build on Crystal 0.29.0

`./script/setup` fails with the following output:

```
  Postinstall make bin
  Failed make bin:
  /usr/local/bin/shards build
  Dependencies are satisfied
  Building: ameba
  Error target ameba failed to compile:
  Error in src/cli.cr:1: while requiring "./ameba/cli/cmd"
  
  require "./ameba/cli/cmd"
  ^
  
  in src/ameba/cli/cmd.cr:1: while requiring "../../ameba"
  
  require "../../ameba"
  ^
  
  in src/ameba.cr:1: while requiring "./ameba/*"
  
  require "./ameba/*"
  ^
  
  Syntax error in src/ameba/inline_comments.cr:97: unexpected token: =
  
  commented? = false
  ^
  
  
  make: *** [bin/ameba] Error 1
```

To fix this, Ameba needs to be upgraded to version 0.10.0. That version, however, does not build on Docker with Crystal version 0.28.0.

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM crystallang/crystal:0.28.0
+FROM crystallang/crystal:0.29.0
 
 RUN apt-get update && \
   apt-get install -y libgconf-2-4 build-essential curl libreadline-dev libevent-dev libssl-dev libxml2-dev libyaml-dev libgmp-dev git  && \

--- a/shard.yml
+++ b/shard.yml
@@ -1,7 +1,7 @@
 name: lucky
 version: 0.13.3
 
-crystal: 0.27.2
+crystal: 0.29.0
 
 authors:
   - Paul Smith <paulcsmith0218@gmail.com>
@@ -60,7 +60,7 @@ dependencies:
 development_dependencies:
   ameba:
     github: veelenga/ameba
-    version: ~> 0.9.0
+    version: ~> 0.10.0
 
 scripts:
   postinstall: script/precompile_tasks

